modules/sticky-table-headers/main/index.coffee

```diff
@@ -43,6 +43,21 @@ cloneEvents = (elem, clone) ->
       cloneChildren = clone.childNodes
       cloneEvents elemChildren[i], cloneChildren[i] for i in [0..elemChildren.length]
 
+getChildrenFromTable = (t, body, single) ->
+  realParents = getChildren(t.select(if body then 'tbody' else 'thead'), 'tr')
+  hx.flatten realParents.map (parent) -> getChildren(hx.select(parent), 'th, td', single)
+
+getChildren = (parent, selector, single) ->
+  children = if single
+    parent.select selector
+  else
+    parent.selectAll selector
+  children.filter((node) -> node.node().parentNode is parent.node()).nodes
```

Comment:
`getChildren` is a basic implementation of logic that could be used for #104 - although I'm not sure how useful this is